test/powershell/Modules/Microsoft.PowerShell.Core/Job.Tests.ps1

```diff
@@ -166,3 +166,63 @@ Describe "Debug-job test" -tag "Feature" {
         $result.Command | Should be "<ScriptBlock>"
     }
 }
+
+Describe "Ampersand background test" -tag "CI" {
+    Context "Simple background job" {
+        AfterEach {
+            Get-Job | Remove-Job -force
+        }
+        It "Background with & produces a job object" {
+            $j = write-output Hi &
+            $j | Should BeOfType System.Management.Automation.Job
+        }
+    }
+    Context "Variable tests" {
+        AfterEach {
+            Get-Job | Remove-Job -force
+        }
+        It "doesn't cause error when variable is missing" {
+            Remove-Item variable:name -erroraction Ignore
+            $j = write-output "Hi $name" &
+            rcjb $j -wait | Should Match "Hi "
+        }
+        It "Copies variables to the child process" {
+            $n1 = "Bob"
+            $n2 = "Mary"
+            ${n 3} = "Bill"
+            $j = write-output "Hi $n1! Hi ${n2}! Hi ${n 3}!" &
+            rcjb $j -wait | Should Match "Hi Bob! Hi Mary! Hi Bill!"
+
+        }
+        It "doesn't overwrite the PID in the child process" {
```

Comment:
This title is difficult to understand. Maybe better "Background job don't overwrite the parent process 'Pid'"